tools/test-rules/frontend-test.md

```
@@ -0,0 +1,424 @@
+# Phase-specific execution
+@agent-test-auditor "execute RED phase - test definition"
+@agent-test-auditor "execute GREEN phase - test implementation"
+@agent-test-auditor "execute REFACTOR phase - optimization"
+```
+
+---
+
+## 🧪 COMPONENT TESTING PATTERNS
+
+### **File Structure (KISS)**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)